src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -59,68 +66,112 @@ class PaymentInformation extends React.Component {
     }
 
     const { paymentInformation } = this.props;
+    const directDepositNotSetup = !get(
+      'responses[0].paymentAccount.accountNumber',
+      paymentInformation,
```

Comment:
According to the info I have, if the user is eligible for direct deposit but no data is found via PPIU, we still receive `responses[0].paymentAccount`, but all of its properties will be empty. So, `responses[0].paymentAccount.accountNumber` should be safe to access directly, but I think `get` makes it more robust.